tests/integ/modin/index/test_index_methods.py

```diff
@@ -39,12 +39,35 @@ def test_index_astype(native_index):
     assert repr(snow_index) == repr(native_index)
 
 
-@sql_count_checker(query_count=3)
+@sql_count_checker(query_count=4)
```

Comment:
is the query count increase caused due to the new assertion added? what is the new assertion try to do there, verify name setup? i thought we decided to delay the support of configuration of that field

Reply:
I rewrote this test and it is fewer queries now